processes/ingest/hathi_trust.py

```diff
@@ -72,7 +72,15 @@ def importFromHathiTrustDataFile(self, fullDump=False):
         )
 
         for hathiFile in fileJSON:
+            print(hathiFile)
```

Comment:
nit: just remember to remove this print statement before we merge in

Reply:
Will do